testutils/zk_utils.go

```diff
@@ -53,7 +67,13 @@ func StartZookeeper() (*ZkControl, error) {
 		cleanup()
 		return nil, errors.Wrap(err, "could not inspect container")
 	}
-	addr := info.NetworkSettings.IPAddress + ":2181"
+
+	var addr string
+	if runtime.GOOS == "darwin" {
+		addr = "127.0.0.1:2181"
```

Comment:
We bind to the host `0.0.0.0` in the port binding configuration above, but here we're using `127.0.0.1`.  Should we be using `0.0.0.0` in both places?

Reply:
In fact, the `HostIP` is the binding address, `0.0.0.0` means that it can be access using any IP of the host. The `127.0.0.1` is connect address, it cannot be `0.0.0.0`